test/e2e-networking-library.sh

```diff
@@ -103,7 +103,11 @@ function net_istio_file_url() {
   local profile="istio-ci-no-mesh"
 
   if (( KIND )); then
-    profile="istio-kind-no-mesh"
+    if (( AMBIENT )); then
```

Comment:
It seems we are getting a lot of env variations here (including the encryption stuff). WDYT about having the istio profile as a variable instead of having `MESH` and `AMBIENT`?